### DISCUSSION TITLE
Better game console accessibility

### DISCUSSION THREAD

**Discussion by Sonicscream** [piqueserver/piqueserver#686] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
When using a process daemon, you lose the ability to interact with the game console.

**Describe the solution you'd like**
The game console should be accessible even while the instance is running via a process daemon (think systemd, etc).

**Describe alternatives you've considered**
Not using a process daemon.

**Additional context**
n/a

**Comment by NotAFile** [comment]
This essentially already exists using IRC. But of course, usage of IRC isn't very high these days. And being able to run commands via an API or similar would be neat either way.

also related: https://github.com/piqueserver/piqueserver/issues/304
